core/src/main/java/cz/xtf/core/bm/BinaryBuild.java

```diff
@@ -56,7 +28,7 @@
 
 @Slf4j
 public abstract class BinaryBuild implements ManagedBuild {
-	private static final String CONTENT_HASH_LABEL_KEY = "xtf.bm/content-hash";
+	static final String CONTENT_HASH_LABEL_KEY = "xtf.bm/content-hash";
```

Comment:
what is the reason for this change?

Reply:
Hash code is computed in subclasses now. I will change it to `protected`.